File: system-test/dotnet/SystemTests/Latest/SmokeTests/System/ShopSmokeTest.cs
using SystemTests.Latest.Base;
using Optivem.Testing;
using Xunit;

namespace SystemTests.Latest.SmokeTests.System;

public class ShopSmokeTest : BaseScenarioDslTest
{
    [Theory]
    [ChannelData(ChannelType.UI, ChannelType.API)]
    public async Task ShouldBeAbleToGoToShop(Channel channel)
    {
        await Scenario(channel).Assume().Shop().ShouldBeRunning();
    }
}

File: system-test/java/src/test/java/com/optivem/shop/systemtest/latest/smoke/system/ShopSmokeTest.java
package com.optivem.shop.systemtest.latest.smoke.system;

import com.optivem.shop.systemtest.latest.base.BaseScenarioDslTest;
import com.optivem.shop.testkit.channel.ChannelType;
import com.optivem.testing.Channel;
import org.junit.jupiter.api.TestTemplate;

class ShopSmokeTest extends BaseScenarioDslTest {
    @TestTemplate
    @Channel({ChannelType.UI, ChannelType.API})
    void shouldBeAbleToGoToShop() {
        scenario.assume().shop().shouldBeRunning();
    }
}

